### ISSUE TITLE
Unable to write proparse.schema file

### ISSUE THREAD

**Issue by samixchoumi** [Riverside-Software/sonar-openedge#248]
I'm making a Jenkins Job, when i use only one dir it's working normaly, but when i put two dir, i got this error, it's not possible to put more than one dir ?
```
[sonar:sonar] 15:55:48.065 INFO  - Using PROPATH : null
[sonar:sonar] 15:55:48.065 INFO  - Using schema : C:/Hudson/workspace/MBUILD_NIGHTLY_LMO46.06/temp/df/*.df,C:/Hudson/workspace/MBUILD_NIGHTLY_LMO46.06/temp/db/*.df
[sonar:sonar] 15:55:48.095 ERROR - Unable to write proparse.schema file
[sonar:sonar] java.io.FileNotFoundException: C:\Hudson\workspace\MBUILD_NIGHTLY_LMO46.06\scripts\progress_build\C
```

I  just saw a strange thing : 
```
[sonar:sonar] 15:55:47.934 WARN  - File 'C:\User\workspace\BUILD_46.06\temp\src\wmsresiz.p' is ignored. 
It is not located in module basedir 'C:\User\workspace\BUILD_46.06\scripts\progress_build'.
```
The strange thing is that i tell to sonar that the workdir is temps/src/ but he is telling me that's not the basedir  but normaly this error has been patched no ?

**Comment by gquerret** [comment]
There was an issue in 1.9.8 when using absolute paths in `sonar.oe.databases`. Problem is corrected in 2.0.0, which has just been released.

However, can't you use PCTDumpSchema to generate the DF in the work directory, and then access the file with a relative path ?

Basedir should be where you checkout your project, and sonar.sources is usually set to 'src' (or close to that).

**Comment by samixchoumi** [comment]
PCTDumpSchema is already used to generate all the df in temp/df and in temp/db, so the path used is a relative path, i tried the 2 solution but i will tell you if the problem is still here with the 2.0.0 update.
